test/test_client.py

```diff
@@ -113,27 +113,26 @@ def test_make_offer(self):
         response = client.make_offer([my_asset], [partner_asset], partner_id, 'TESTOWA OFERTA')
         self.assertIsNotNone(response)
 
-
-    def test_make_offer_url(self):
-        sample_trade_url="https://steamcommunity.com/tradeoffer/new/?partner=314218906&token=sgA4FdNm"  # a sample trade url with escrow time of 15 days cause mobile auth not added
+    def test_make_offer_with_trade_url(self):
+        sample_trade_url="https://steamcommunity.com/tradeoffer/new/?partner=314218906&token=sgA4FdNm"
```

Comment:
I asked why is the token parameter here, what is for and if it is a part of trade_url?